CHANGELOG.md

```diff
@@ -2,7 +2,14 @@
 
 ## 0.1 DeepGNN
 
-### 0.1.52 - 2022-07-27
+### [Unreleased]
+
+### [0.1.53] - 2022-08-02
+
+#### Fixed
+- json/tsv converter didn't sort edges by types resulted in incorrect sampling.
```

Comment:
This is in a different format than the other section of the change log.

Reply:
Reformatted previous messages.